server/profileFormSubmission.js

```diff
@@ -0,0 +1,35 @@
+const { Pool } = require('pg');
+const { allowedNodeEnvironmentFlags } = require('process');
+const pool = new Pool({ 
+    user: process.env.user,
+    host: process.env.host,
+    database: process.env.database,
+    password: process.env.password,
+    port: process.env.port,
+});
+
+
+module.exports.profileInsertion = async (event, callback )=> {
+    const body = JSON.parse(event.body);
+    const text = 'INSERT INTO cleaner_profile(first_name, last_name, contact_num) VALUES($1, $2, $3) RETURNING *'
+    const values = [body.firstName, body.lastName, body.number]
```

Comment:
Are we doing anything with the references rn, or is that out of scope for this story?

Reply:
nothing for the references here